README.md

```diff
@@ -80,14 +81,16 @@ optional arguments:
   -l, --loop            loop a single video; otherwise play through the files
                         in the videos directory
   -o {DEBUG,INFO,WARNING,ERROR,CRITICAL}, --loglevel {DEBUG,INFO,WARNING,ERROR,CRITICAL}
-                         minimum importance-level of messages displayed and
-                         saved to the logfile (default: INFO)
+                        minimum importance-level of messages displayed and
+                        saved to the logfile (default: INFO)
+  -S, --subtitles       Display SRT subtitles
+  -t, --timecode        Display video timecode
 
 Args that start with '--' (eg. -f) can also be set in a config file
 (slowmovie.conf). Config file syntax allows: key=value, flag=true,
-stuff=[a,b,c] (for details, see syntax at https://pypi.org/project/ConfigArgParse).
-If an arg is specified in more than one place, then commandline values override
-config file values, which in turn override defaults.
+stuff=[a,b,c] (for details, see syntax at https://goo.gl/R74nmi). If an arg is
```

Comment:
Should this stay a shortlink? I remember @missionfloyd changing it [in a past commit](3e1891ebcf2d876cdaac0350e3c1ce65dc703e8c) and I agree that the longer link is nicer because it shows where it's leading. But the actual output of `slowmovie.py -h` *does* have the shortlink in it. Thoughts one way or the other?

Reply:
I forgot about that. I just copied the entire output.